lib/main.dart

```diff
@@ -465,11 +498,39 @@ class _CardInterfaceState extends State<CardInterface> {
         setState(() => _showVerseNumbers = v);
       };
     }
-    return SwitchListTile(
-      title: Text('Show Verse Numbers'),
-      value: _showVerseNumbers,
-      onChanged: onChanged,
-      dense: true,
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        Container(
+          margin: EdgeInsets.only(right: 15.0, left: 20.0),
+          child: Text(
+            "Verse Numbers",
+            style: TextStyle(
+              color: _verseStatus != VerseStatus.read
```

Comment:
I would suggest making `_verseStatus != VerseStatus.read` a method like `inReadMode()` and using that across your methods